src/agent/include/register.hpp

```diff
@@ -1,8 +1,16 @@
 #pragma once
 
+#include <http_client.hpp>
+
+#include <boost/beast/core.hpp>
```

Comment:
I'm not sure that we need both core and http in the header file.

Reply:
Fixed!